src/products/android-l10n/adding_projects.md

```diff
@@ -0,0 +1,19 @@
+# Adding a new project to android-l10n
+
+Follow the steps below to add new projects to [android-l10n](https://github.com/mozilla-l10n/android-l10n). You will have to coordinate with android-l10n developers and a technical Localization EPM throughout the process.
+
+## Setting up the project in source and l10n repositories
+
+Before anything happens, check that the new project is properly localizable by following the directions in [this section](https://github.com/mozilla-l10n/documentation/blob/main/src/tools/pontoon/adding_new_project.md#adding-a-new-project-on-pontoon). Once the new project exists in its corresponding source repo and is properly localizable, it can then be added to the `android-l10n` repository.
+
+An Android developer should create a TOML file in their code repository and under it's corresponding project folder (example with Focus [here](https://github.com/mozilla-mobile/firefox-android/tree/main/focus-android)), as well as as a file containing the source strings (example with Focus [here](https://github.com/mozilla-mobile/firefox-android/blob/c718159480512abf370abcfd5aaf54b1d81074d3/focus-android/app/src/main/res/values/strings.xml)). The `firefox-android` repository contains all [current android projects](https://github.com/mozilla-mobile/firefox-android/tree/main).
```

Comment:
```suggestion
Android developers need to create a TOML file in their code repository and under its corresponding project folder (example for Focus [here](https://github.com/mozilla-mobile/firefox-android/blob/main/focus-android/l10n.toml)), as well as as the file containing the source strings in `res/values/strings.xml` (example with Focus [here](https://github.com/mozilla-mobile/firefox-android/blob/c718159480512abf370abcfd5aaf54b1d81074d3/focus-android/app/src/main/res/values/strings.xml)). The `firefox-android` repository contains all [current android projects](https://github.com/mozilla-mobile/firefox-android/tree/main).
```

Reply:
What about the list of locales in the config file? Are there limitations?

Reply:
I don't believe there are any limitations, that I can remember (vs iOS that does)
Is that what you were pointing out to call out - or am I missing something? 🤔

Reply:
Yes, but also that the TOML file needs to have some locales listed to start with.

Reply:
Ah, right. IIRC we announce a new project and let localizers opt-in beforehand (through our communication channels), and so that we have a list of locales to start with. We also need to check that they are already contained in a-c TOML, and otherwise add them there.
I think that covers it?

Reply:
(if a-c is required, which might not be a given) But yes, that sounds right.

Reply:
Thanks for confirming, will make fixes